include/seastar/rpc/rpc.hh

```diff
@@ -203,7 +203,11 @@ class logger {
 
     // _seastar_logger will always be used first if it's available
     template <typename... Args>
+#if FMT_VERSION >= 80000
+    void log(log_level level, fmt::format_string<Args...> fmt, Args&&... args) const {
+#else
```

Comment:
I'm not very happy with this flood of #if-s all over the header. Is it possible to do smth like
```c++
#if FMT_VERSION >= 80000
template <typename... Args>
using fmt_string = fmt::format<Args...>;
#else
template <typename... Args>
using fmt_string = const char* fmt;
#endif
```
once and use the `typename fmt_string<Args...>` here and below?

Reply:
@xemul hi Pavel, thank you very much. love this. i assume you meant to change `log.hh`, so i incorporated your change into that file instead.